docs/intro-to-eda.md

```diff
@@ -1,56 +1,53 @@
 ---
-title: Learn About Event Driven Architecture
+title: About Event Driven Architecture at VA
 ---
 
 # Introduction
-The Enterprise Event Bus is an asynchronous event processing system that spans systems and lines of business at the VA. Event-driven architecture uses events, which are specific instances of something that happened, to communicate with systems that are subscribed to the stream of events. The systems that are producing the events [producers](https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/produce-events/), are decoupled from the systems that are consuming their events [consumers](https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/consume-events/).
+The Enterprise Event Bus is an asynchronous event processing system that spans systems and lines of business at VA. Event-driven architecture uses events – types of actions, such as a Veteran creating a medical appointment or updating their beneficiaries – to communicate with systems that are subscribed to the stream of events. The systems that are producing the events, also known as [producers](https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/produce-events/), are decoupled from the systems that are consuming their events, also known as [consumers](https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/consume-events/).
 
-See this [brief video](https://www.youtube.com/watch?v=R6tUoxx2gVY) for a quick overview of event-driven architecture.
-
-_The following diagram illustrates how the different components of the Enterprise Event Bus come together to produce streams of events._
+_The following diagram illustrates how the different components of the Enterprise Event Bus come together to produce streams of events. Open the diagram in [Github](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Diagrams/future%20state%20whole.png)._
 
```

Comment:
Instead of the link I added in the Google doc, let's instead say `[View the full-sized diagram.](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/95644573/f0dfe62a-8509-459c-bd9a-074e0babb22b)`

Reply:
Updated. Marking as resolved.